plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -693,74 +693,38 @@ def deserialize(self, serialized, file_name = None):
                         if buildplate_id is None:
                             continue
 
-                        from cura.Machines.VariantManager import VariantType
                         variant_manager = CuraApplication.getInstance().getVariantManager()
                         variant_node = variant_manager.getVariantNode(machine_id, buildplate_id,
                                                                       variant_type = VariantType.BUILD_PLATE)
                         if not variant_node:
                             continue
 
-                        buildplate_compatibility = machine_compatibility
-                        buildplate_recommended = machine_compatibility
-                        settings = buildplate.iterfind("./um:setting", self.__namespaces)
-                        for entry in settings:
-                            key = entry.get("key")
-                            if key in self.__unmapped_settings:
-                                if key == "hardware compatible":
-                                    buildplate_compatibility = self._parseCompatibleValue(entry.text)
-                                elif key == "hardware recommended":
-                                    buildplate_recommended = self._parseCompatibleValue(entry.text)
-                            else:
-                                Logger.log("d", "Unsupported material setting %s", key)
+                        _, buildplate_unmapped_settings_dict = self._getSettingsDictForNode(buildplate)
```

Comment:
_, - is very strange name for variable

Reply:
That means you wanna ignore that variable (but have no control over the fact that it comes back from the called method). It's actually a very common practice in tuples or method args.